src/ethaniccc/Mockingbird/listener/MockingbirdListener.php

```diff
@@ -29,7 +29,7 @@
 use pocketmine\network\mcpe\protocol\PlayerAuthInputPacket;
 use pocketmine\network\mcpe\protocol\ProtocolInfo;
 use pocketmine\network\mcpe\protocol\SetActorMotionPacket;
-use pocketmine\network\mcpe\protocol\StartGamePacket;
+use pocketmine\network\mcpe\protocol\GamePacket;
```

Comment:
You are wrong, the packet is called `StartGamePacket`

Reply:
Okay